### PR TITLE
Feat(Timeline) - Waviez Edition

### PR DESCRIPTION
**Pull request by Pongpon00**
### Changelog
- Add new Timeline feature with a progress bar.
- Update App.tsx for testing
- Upload new assets

### CODE REVIEW

File: public/LandingBackground.jpg
Comment (sokungz01) [inline public/LandingBackground.jpg:1]: Can you convert the image files (jpg/png) to webp file?
These image files will cause the website load time.

**Comment by sokungz01** [comment]
Can you fix this white box on the mobile?

Screenshot 2567-07-29 at 19 08 48

**Comment by Pongpon00** [comment]
ok pap

**Review by sokungz01 — CHANGES_REQUESTED** [review]

**Comment by Pongpon00** [comment]
I already fix the responsive design
timeline responsive design

File: public/LandingBackground.jpg
Comment (Pongpon00) [reply public/LandingBackground.jpg:1]: I tried to convert the jpg image to webp image but it doesn't reduce the file size. Instead, the file size is bigger.  

image

Comment (Pongpon00) [reply public/LandingBackground.jpg:1]: do you have any suggest?

Comment (sokungz01) [reply public/LandingBackground.jpg:1]: Can you ask @SSSBoOm ?

Comment (Pongpon00) [reply public/LandingBackground.jpg:1]: Ok

Comment (Pongpon00) [reply public/LandingBackground.jpg:1]: Problem resolved!

**Comment by sokungz01** [comment]
LGTM ✨

**Comment by sokungz01** [comment]
@Pongpon00 Can you resolve a conflict, please?

**Comment by sokungz01** [comment]
> @Pongpon00 Can you resolve a conflict, please?

lmk, if you solved a conflict

**Comment by Pongpon00** [comment]
Ok, Got it.

**Pull request merged**